cfgov/jinja2/v1/owning-a-home/_templates/form-explainer.html

```diff
@@ -0,0 +1,235 @@
+                <div class="form-explainer_page-buttons">
+                    <button class="prev a-btn a-btn__disabled">
+                        <span class="u-visually-hidden">Prev page</span>
+                        <span class="cf-icon cf-icon-left"></span>
+                    </button>
+                    <button class="a-btn next">
+                        <span class="u-visually-hidden">Next page</span>
+                        <span class="cf-icon cf-icon-right"></span>
+                    </button>
+                  </div>
+                <img class="image-map_image"
+                     src="{{ page.img }}"
+                     alt="">
+                {{ render_map_points( page.terms, loop.index ) }}
+            </div>
+        </div>
+        <figcaption class="terms explain_terms">
+            <div class="o-expandable-group
+                        o-expandable-group__form-explainer
+                        o-expandable-group__accordion">
+                {{ render_terms( page.terms, loop.index ) }}
+            </div>
+        </figcaption>
+    </figure>
+    {% endfor %}
+</div><!-- ./explain -->
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_map_points( terms, page_num )
+
+   ==========================================================================
+
+   Description:
+
+   Render a list of o-expandable terms with definitions.
+
+   terms:              An array of term objects. See item() for info on term
+                       objects.
+
+   page_num:           The page that these map points belong to.
+
+   ========================================================================== #}
+
+{% macro render_map_points( terms, page_num ) %}
+{% for item in terms %}
+<a class="image-map_overlay image-map_overlay__{{ item.category }}"
+     style="{{ 'top:' + item.top + ';' 'left:' + item.left + ';' }}{{ 'width:' + item.width + ';' 'height:' + item.height + ';' if item.width and item.height }}"
+     href="{{ '#page-' + page_num|string + '-' + item.category + '-' + item.id }}"
+     id="{{ 'image-map_overlay__' + page_num|string + '-' + item.category + '-' + item.id }}"
+     tabindex="-1">
+    <span class="u-visually-hidden">{{ item.term }}</span>
+ </a>
+{% endfor %}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_terms( terms, page_num )
+
+   ==========================================================================
+
+   Description:
+
+   Render a list of o-expandable terms with definitions.
+
+   terms:              An array of term objects. See item() for info on term
+                       objects.
+
+   page_num:           The page that these terms belong to.
+
+   ========================================================================== #}
+
+{% macro render_terms( terms, page_num ) %}
+{% for item in terms %}
+    {{ term( item, loop.index, page_num ) }}
```

Comment:
Oh I see why you didn't name it `term`. Perhaps `render_term(…` to match format elsewhere.

Reply:
I changed the other macro name to `render_term`, so I modify the name.